src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -1040,6 +1040,55 @@ public enum KeyContainerPermissionFlags
         ChangeAcl = 8192,
         AllFlags = 13111,
     }
+    public sealed partial class MediaPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
+    {
+        public MediaPermission() { }
+        public MediaPermission(System.Security.Permissions.MediaPermissionAudio permissionAudio) { }
+        public MediaPermission(System.Security.Permissions.MediaPermissionAudio permissionAudio, System.Security.Permissions.MediaPermissionVideo permissionVideo, System.Security.Permissions.MediaPermissionImage permissionImage) { }
+        public MediaPermission(System.Security.Permissions.MediaPermissionImage permissionImage) { }
+        public MediaPermission(System.Security.Permissions.MediaPermissionVideo permissionVideo) { }
+        public MediaPermission(System.Security.Permissions.PermissionState state) { }
+        public System.Security.Permissions.MediaPermissionAudio Audio { get { throw null; } }
+        public System.Security.Permissions.MediaPermissionImage Image { get { throw null; } }
+        public System.Security.Permissions.MediaPermissionVideo Video { get { throw null; } }
+        public override System.Security.IPermission Copy() { throw null; }
```

Comment:
perhaps call `base.Copy()`?

Reply:
This is for the ref assembly. I don't think we want any actual implementation here, but I'll map these comments to the actual implementation file.

Reply:
ah yes - thanks for the catch!